_posts/en/posts/2025-06-03-relay-statement.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Bitcoin Core development and transaction relay policy
+permalink: /en/2025/06/03/relay-statement/
```

Comment:
nit: could bump to the (assumed) merging/publishing date and rename file accordingly
```suggestion
permalink: /en/2025/06/05/relay-statement/
```